code/game/objects/items/devices/radio/headset.dm

```diff
@@ -290,6 +290,30 @@ GLOBAL_LIST_INIT(channel_tokens, list(
 /obj/item/radio/headset/silicon/can_receive(freq, level)
 	return ..(freq, level, TRUE)
 
+/obj/item/radio/headset/clown
+	name = "Clowning Headset"
+	desc = "A headset with a clowning filter, honk!"
+	icon_state = "honk_headset"
+	var/honken = TRUE
+
+/obj/item/radio/headset/clown/talk_into_impl(atom/movable/M, message, channel, list/spans, datum/language/language, list/message_mods)
+	if(honken)
+		spans |= SPAN_CLOWN
+	. = ..()
+
+/obj/item/radio/headset/clown/AltClick(mob/user)
```

Comment:
This should use the existing TGUI volume functionality instead of Alt+Click.

Reply:
i have 0 knowledge of how to use tgui, youll have to tell me how

Reply:
you know how as command, AI, etc you can open the radio and check "Subspace Tx" or "High Volume Mode"

add a button for that, but for honk (high volume) mode.

look at the command radios for examples.